go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containerd/containerd/v2
 
-go 1.23.0
+go 1.24
```

Comment:
I still have to look into the NRI side of changes. But this will force the minimal version to 1.24. Is it possible to keep the go.mod version at 1.23 in NRI and toolchain at 1.24?